Update Mvvmcross

AntonTereshko1995 opened the pull request.

Commit:
Update Mvvmcross

alexshikov commented on Mvx.Geocoder/Platforms/Ios/Plugin.cs:
```diff
@@ -0,0 +1,18 @@
+
+namespace Mvx.Geocoder.Platforms.Ios
+{
+	namespace Mvx.Geocoder.Platforms.Android
```
unnecessary namespace

alexshikov replied:
This namespace is unnecessary

alexshikov commented on Mvx.Geocoder.Sample.Core/App.cs:
```diff
@@ -1,18 +1,18 @@
+			RegisterAppStart<FirstViewModel>();
+		}
+	}
+}
```
It is important to keep the author's code formatting (in that case spaces should not be changed to the tabs).

(no need to fix that in that case, but please be careful in other PRs)

alexshikov commented on Mvx.Geocoder.Sample.Core/ViewModels/FirstViewModel.cs:
```diff
@@ -1,75 +1,67 @@
 		#endregion
 
-		private readonly IGeocoder geocoder;
+		private readonly IGeocoder _geocoder;
```
Same here with formatting... (no need to fix that)

alexshikov reviewed (changes requested):
Let's keep the new code formatting. But overall, that must not be changed.

Commit:
Fix namespace and formatting

alexshikov committed:
[iOS] Use new Mvx approach to initialize App

alexshikov committed:
Ignore .vs folder

alexshikov committed:
Update nuget spec (still used old-school approach)

We get an error while trying to build nugets from VS (on Mac and Win):
error MSB4044: The “FilterAssemblies” task was not given a value
for the required parameter “DesignTimeBuild”.

Decided to keep using nuspec and scripts to pack/push packages.
As soon as we can use VS - the scripts should be removed.
